Week2/Homework/Grade-calculator.js

```diff
@@ -0,0 +1,48 @@
+ 
+
+  let score=prompt("Enter your score"); // Manual input
+  let full_Score=100;
+
+ // This function turns scores into percentage values.
+ function percentage_calculator (studentScore, maxScore){
+    let studentscore = Math.round(( score / full_Score) * 100);
+    return studentscore;
+}
+
+
+// This function shows the letter grade and score as percentage value.
+
+function grade_calculator (studentscore , studentGrade){
+ 
+    if(studentscore >= 90) {
```

Comment:
I would suggest putting a check before this to make sure that the value entered by the user is a `number` type and is between 0 and 100. Since you are using `prompt`, the user can also enter some garbage values which would result in your code not working as expected 🙂

Reply:
Fixed.